python/src/aiconfig/editor/client/src/components/prompt/PromptActionBar.tsx

```diff
@@ -44,48 +45,60 @@ export default memo(function PromptActionBar({
   const promptMetadataSchema = promptSchema?.prompt_metadata;
 
   return (
-    <Flex direction="column" justify="space-between">
+    <Flex direction="column" justify="space-between" h="100%">
       {isExpanded ? (
-        <Container miw="400px">
-          <ActionIcon size="sm" onClick={() => setIsExpanded(false)}>
-            <IconClearAll />
-          </ActionIcon>
-          <Tabs defaultValue="settings">
-            <Tabs.List>
-              <Tabs.Tab value="settings">Settings</Tabs.Tab>
-              {checkParametersSupported(prompt) && (
-                <Tabs.Tab value="parameters">
-                  Local Variables (Parameters)
-                </Tabs.Tab>
-              )}
-            </Tabs.List>
+        <>
+          <Container miw="400px">
+            <ActionIcon
+              size="sm"
+              onClick={() => setIsExpanded(false)}
+              mt="0.5em"
+            >
```

Comment:
nit: I think having an "x" button would be more intuitive. Also having it inline horizontally with the tabs instead of ontop would look cleaner. Again not a blocker for MVP